packages/angular/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
Is `src/app` really necessary?

Reply:
Perhaps this should just have a `projects` folder? Similar to `--create-application=false` when running `ng new`?

Reply:
`app` is moved to `projects/angular-app`